spring-cloud-huawei-swagger/src/main/java/com/huaweicloud/swagger/ServiceCombSwaggerHandlerImpl.java

```diff
@@ -100,13 +125,13 @@ public void init(String appName, String serviceName) {
   }
 
   /**
-   * schema注册调用接口可以改为异步,在和java-chassis组网场景下需要同步加载 todo: schema生成理论上也可改为异步，但基于spring-fox，要看下能否实现
+   * todo: schema generate also can be async , use aop around method
    *
    * @param microserviceId
    * @param schemaIds
    */
   public void registerSwagger(String microserviceId, List<String> schemaIds) {
-    if (isSync) {
+    if (withJavaChassis) {
```

Comment:
Do you mean a different configuration? This is "if register schemas asynchronousely"

Reply:
done